src/main/java/org/broadinstitute/consent/http/db/mapper/UserWithRolesReducer.java

```diff
@@ -56,34 +48,56 @@ public void accumulate(Map<Integer, User> map, RowView rowView) {
         }
       }
     } catch (MappingException e) {
-      //Ignore institution mapping errors, possible for new users to not have an institution
+      logDebug("Error adding Institution to User: %s".formatted(e.getMessage()));
     }
     //user role join can cause duplication of data if done in tandem with joins on other tables
     //ex) The same LC can end up being repeated multiple times
     //Below only adds LC if not currently saved on the array
     try {
       if (rowView.getColumn("lc_id", Integer.class) != null) {
-        int lcId = rowView.getColumn("lc_id", Integer.class);
-        LibraryCard lc;
-        Optional<LibraryCard> existingLibraryCard = user.getLibraryCards().stream()
-                .filter(card -> card.getId().equals(lcId))
-                .findFirst();
-        lc = existingLibraryCard.orElseGet(() -> rowView.getRow(LibraryCard.class));
+        LibraryCard lc = rowView.getRow(LibraryCard.class);
         if (rowView.getColumn("lc_daa_id", Integer.class) != null) {
           lc.addDaa(rowView.getColumn("lc_daa_id", Integer.class));
         }
-        user.addLibraryCard(lc);
+        user.setLibraryCard(lc);
       }
     } catch (MappingException e) {
-      //Ignore exceptions here, user may not have a library card issued under this instiution
+      logDebug("Error adding Library Card to User: %s".formatted(e.getMessage()));
     }
     try {
       if (Objects.nonNull(rowView.getColumn("up_property_id", Integer.class))) {
         UserProperty p = rowView.getRow(UserProperty.class);
         user.addProperty(p);
       }
     } catch (MappingException e) {
-      // Ignore any attempt to map a column that doesn't exist
+      logDebug("Error adding User Property to User: %s".formatted(e.getMessage()));
+    }
+  }
+
+  // Some queries look for `user_role_id` while those that use a prefix look for `u_user_role_id`
+  private UserRole mapUserRoleFromRowView(RowView rowView, Integer userId) {
```

Comment:
In theory, I don't think this method should be necessary. However, I found that the mapping wasn't working for cases where the prefix was `ur_` and couldn't get that to work like I think it should.